ddtrace/contrib/unittest/patch.py

```diff
@@ -561,8 +569,23 @@ def _start_test_span(instance, test_suite_span: ddtrace.Span) -> ddtrace.Span:
     Starts a test  span and sets the required tags for a `unittest` test instance.
     """
     tracer = getattr(unittest, "_datadog_tracer", _CIVisibility._instance.tracer)
-    test_suite_name = _extract_suite_name_from_test_method(instance)
     test_name = _extract_test_method_name(instance)
+    test_method_object = _extract_test_method_object(instance)
+
+    source_file_path = None
+    start_line = None
+    end_line = None
+
+    if test_method_object:
```

Comment:
This logic appears to be duplicated between the pytest and unittest integrations. You might consider putting it into a function in `ddtrace.contrib.test_utils` or a similar appropriate place.

Reply:
Good point, this has been addressed now.